File: data/transcoder_evaluation_gfg/c/PROGRAM_FIND_REMAINDER_LARGE_NUMBER_DIVIDED_11.c
// Copyright (c) 2019-present, Facebook, Inc.
// All rights reserved.
//
// This source code is licensed under the license found in the
// LICENSE file in the root directory of this source tree.
//


#include <stdio.h>
#include <math.h>
#include <stdlib.h>
#include <limits.h>
#include <stdbool.h>

int min(int x, int y) { return (x < y)? x: y; }
int max(int x, int y) { return (x > y)? x: y; }
int cmpfunc (const void * a, const void * b) {return ( *(int*)a - *(int*)b );}
int len (int arr [ ]) {return ((int) (sizeof (arr) / sizeof (arr)[0]));}
void sort (int arr [ ], int n) {qsort (arr, n, sizeof(int), cmpfunc);}

int f_gold ( char str [] ) {
  int len = strlen(str);
  int num, rem = 0;
  for ( int i = 0;
  i < len;
  i ++ ) {
    num = rem * 10 + ( str [ i ] - '0' );
    rem = num % 11;
  }
  return rem;
}


int f_filled ( char str [] ) {}

int main(void) {
    int n_success = 0;
    char param0[][100] = {"DvsNZVNZ","1170","10","evsPwREbSY","09219178704","1001010","SkZbWSajDKmiG","0287976763","011011000111","lUn"};
    for(int i = 0; i < len(param0); ++i)
    {
        if(f_filled(param0[i]) == f_gold(param0[i]))
        {
            n_success+=1;
        }
    }
    printf("#Results:", " ", n_success, ", ", len(param0));
    return 0;
}

File: data/transcoder_evaluation_gfg/go/PROGRAM_FIND_REMAINDER_LARGE_NUMBER_DIVIDED_11.go
package main

import (
		"fmt"
	"os"
	"unsafe"
)

func min(x int, y int) int {
	if x < y {
		return x
	}
	return y
}
func max(x int, y int) int {
	if x > y {
		return x
	}
	return y
}
func cmpfunc(a unsafe.Pointer, b unsafe.Pointer) int {
	return *(*int)(a) - *(*int)(b)
}


func f_gold(str []byte) int {
	var (
		len_ int = int(len(str))
		num  int
		rem  int = 0
	)
	for i := int(0); i < len_; i++ {
		num = rem*10 + int(str[i]-byte('0'))
		rem = num % 11
	}
	return rem
}
//TOFILL
func main() {
	var (
		n_success int           = 0
		param0    [10][]byte = [10][]byte{func() []byte {
			var t []byte
			copy(t[:], ([]byte)("DvsNZVNZ"))
			return t
		}(), func() []byte {
			var t []byte
			copy(t[:], ([]byte)("1170"))
			return t
		}(), func() []byte {
			var t []byte
			copy(t[:], ([]byte)("10"))
			return t
		}(), func() []byte {
			var t []byte
			copy(t[:], ([]byte)("evsPwREbSY"))
			return t
		}(), func() []byte {
			var t []byte
			copy(t[:], ([]byte)("09219178704"))
			return t
		}(), func() []byte {
			var t []byte
			copy(t[:], ([]byte)("1001010"))
			return t
		}(), func() []byte {
			var t []byte
			copy(t[:], ([]byte)("SkZbWSajDKmiG"))
			return t
		}(), func() []byte {
			var t []byte
			copy(t[:], ([]byte)("0287976763"))
			return t
		}(), func() []byte {
			var t []byte
			copy(t[:], ([]byte)("011011000111"))
			return t
		}(), func() []byte {
			var t []byte
			copy(t[:], ([]byte)("lUn"))
			return t
		}()}
	)
	for i := int(0); i < len(param0[:]); i++ {
		if f_filled(param0[i][:]) == f_gold(param0[i][:]) {
			n_success += 1
		}
	}
	fmt.Print("#Results:", " ", n_success, ", ", len(param0[:]))
	os.Exit(0)
}
